api/handler.go

```
@@ -26,7 +27,18 @@ func (s Server) createOrder(c echo.Context) error {
 
-	s.producer.ProduceOrderCreated(order.Id.Hex(), order)
+	event := types.OrderCreatedEvent{
+		Id:              order.Id.Hex(),
+		Type:            "OrderCreated",
+		Source:          "OrderService",
+		SpecVersion:     "1.0",
+		Data:            order,
+		DataContentType: "application/json",
+	}
+
+	s.producer.ProduceOrderCreated(order.Id.Hex(), event)
```

golangci-lint (1.62.2)

Error return value of `s.producer.ProduceOrderCreated` is not checked

(errcheck)